satpy/readers/pmw_channels_definitions.py

```diff
@@ -0,0 +1,415 @@
+
+        """
+        if other is None:
+            return False
+        if isinstance(other, numbers.Number):
+            return other in self
+        if isinstance(other, (tuple, list)) and len(other) == 3:
+            return other in self
+        return super().__eq__(other)
+
+    def __ne__(self, other):
+        """Return the opposite of `__eq__`."""
+        return not self == other
+
+    def __lt__(self, other):
+        """Compare to another frequency."""
+        if other is None:
+            return False
+        return super().__lt__(other)
+
+    def __gt__(self, other):
+        """Compare to another frequency."""
+        if other is None:
+            return True
+        return super().__gt__(other)
+
+    def __hash__(self):
+        """Hash this tuple."""
+        return tuple.__hash__(self)
+
+    def __str__(self):
+        """Format for print out."""
+        return "{0.central} {0.unit} ({0.side}_{0.bandwidth} {0.unit})".format(self)
+
+    def __contains__(self, other):
+        """Check if this double-side-band 'contains' *other*."""
+        if other is None:
+            return False
+        if isinstance(other, numbers.Number):
+            if (self.central + self.side - self.bandwidth/2. <= other
+                    <= self.central + self.side + self.bandwidth/2.):
+                return True
+            if (self.central - self.side - self.bandwidth/2. <= other
+                    <= self.central - self.side + self.bandwidth/2.):
+                return True
+            return False
+
+        if isinstance(other, (tuple, list)) and len(other) == 3:
+            return ((self.central - self.side - self.bandwidth/2. <=
+                     other[0] - other[1] - other[2]/2. and
+                     self.central - self.side + self.bandwidth/2. >=
+                     other[0] - other[1] + other[2]/2.) or
+                    (self.central + self.side - self.bandwidth/2. <=
+                     other[0] + other[1] - other[2]/2. and
+                     self.central + self.side + self.bandwidth/2. >=
+                     other[0] + other[1] + other[2]/2.))
+
+        with suppress(AttributeError):
+            if self.unit != other.unit:
+                raise NotImplementedError("Can't compare frequency ranges with different units.")
+            return ((self.central - self.side - self.bandwidth/2. <=
+                     other.central - other.side - other.bandwidth/2. and
+                     self.central - self.side + self.bandwidth/2. >=
+                     other.central - other.side + other.bandwidth/2.) or
+                    (self.central + self.side - self.bandwidth/2. <=
+                     other.central + other.side - other.bandwidth/2. and
+                     self.central + self.side + self.bandwidth/2. >=
+                     other.central + other.side + other.bandwidth/2.))
+
+        return False
+
+    def distance(self, value):
+        """Get the distance from value."""
```

Comment:
Here too could be some info on how this distance is defined.

Reply:
Added more explanation.